frappe/config/integrations.py

```diff
@@ -72,6 +72,16 @@ def get_data():
 		{
 			"label": _("External Documents"),
 			"items": [
+				{
```

Comment:
maybe make a separate section for `Google Services` ?

Reply:
Sure no problem, I'll do that!